hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowModule.java

```diff
@@ -45,6 +52,23 @@ static Function<SignatureMap, PubKeyToSigBytes> provideKeyToSigFactory() {
     @Binds
     SignaturePreparer bindSignaturePreparer(MonoSignaturePreparer signaturePreparer);
 
+    @Binds
+    SignatureVerifier bindSignatureVerifier(SignatureVerifierImpl signatureVerifier);
+
+    @Binds
+    SignatureExpander bindSignatureExpander(SignatureExpanderImpl signatureExpander);
+
+    // FUTURE: This should be updated to use the new config module and real config loading when possible
+    @Provides
```

Comment:
https://github.com/hashgraph/hedera-services/pull/6291 does it. Maybe you can deprecate your call